src/field_theory/intermediate_field.lean

```diff
@@ -317,6 +321,37 @@ begin
     left, refl }
 end
 
+/-- Let `L/K` be a field extension and let `E, F` be intermediate fields with `E ≤ F`(where
+  `hEF` is a term of type `E ≤ F`). Then `intermediate_field.incusion hEF` is the `K`-algebra
+  homomorphism `E →ₐ[K] F` given by the inclusion. -/
+def inclusion {K L : Type*} [field K] [field L] [algebra K L]
+  {E F : intermediate_field K L} (hEF : E ≤ F):
+  E →ₐ[K] F :=
+{ to_fun := set.inclusion hEF,
+  map_one' := rfl,
+  map_add' := λ _ _, rfl,
+  map_mul' := λ _ _, rfl,
+  map_zero' := rfl,
+  commutes' := λ _, rfl }
+
+lemma inclusion_eq_identity {K L : Type*} [field K] [field L] [algebra K L]
+  {E F : intermediate_field K L} (hEF : E ≤ F) (x : E) :
+  F.val ((intermediate_field.inclusion hEF) x) = E.val x := rfl
```

Comment:
```suggestion
  ↑(intermediate_field.inclusion hEF x) = (x : L) := rfl
```

Reply:
If I make this change (to the lemma now called `subalgebra.coe_inclusion_apply`), then the second line of the proof of `subalgebra.inclusion_mk` breaks. Should I still make this change and try to fix the other proof?

Reply:
That lemma should just be proved by `rfl` anyway